### PR TITLE
WEB-1247: License Updated Message

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** Update css for success message

**JIRA:** https://liv-it.atlassian.net/browse/WEB-1247

**Related PR:** https://github.com/Livit/Livit.Learn.EdxCourseLicenses/pull/17

**Merge deadline:** List merge deadline (if any)

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [x] tag reviewer
- [x] tag reviewer

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [ ] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [x] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

File: lms/templates/labster/course_license.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
     }
     .label {color: #646464}
     .messages {margin: 10px 0 20px}
-    .messages .error {color: red}
+    .messages .error, .messages .success {color: red}
```
Comment (polesye) [inline lms/templates/labster/course_license.html]: @lukyfebrianto why success message is red? Shouldn't it be green?

Comment (lukyfebrianto) [reply lms/templates/labster/course_license.html]: I see the picture on the ticket is red @polesye,  should I change to green ?
CMIIW

Comment (polesye) [reply lms/templates/labster/course_license.html]: yes, please. I think it was a quick mockup

Comment (lukyfebrianto) [reply lms/templates/labster/course_license.html]: updated

**Review by polesye — APPROVED** [review]

**Review by TamoshaytisV — APPROVED** [review]

**Pull request merged**